doc/api/Miscellaneous/MultiThreading.md

```diff
@@ -380,3 +380,32 @@ if (currentModeInfo === null) {
   console.info("We're running completely in main thread.");
 }
 ```
+
+## Note about ES5 Environment
+
+By default, the Worker file will use es2017 features which should be compatible to a
+majority of devices.
+
+However, some older devices might not be compatible with it yet still compatible to the
+WebWorker API (and thus the `MULTI_THREAD` feature), yet not with ES2017. For those
+platforms, the RxPlayer also provide an ES5 variant of the Worker file:
+
+- The easiest way is to just import in your application its "embedded" version, exported
+  through the `"rx-player/experimental/features/embeds"` path:
+
+  ```js
+  import { EMBEDDED_WORKER_ES5 } from "rx-player/experimental/features/embeds";
+  ```
+
+  This allows to bypass the need to store and serve separately that file.
+
+  If you would prefer more control and a smaller bundle size, you may instead consider the
+  other following ways to it as a separate file.
+
+- With every release note published on GitHub as `worker.es5.js` (you should only use the
+  file linked to the RxPlayer's version you're using),
+
+- It is also available as `dist/worker.es5.js` from the root directory of the project
+  published on npm. As such, it might already be found in your project's directory, for
+  example in the `node_modules` directory (most probably in `node_modules/rx-player/dist/`
+  depending on your project).
```

Comment:
We have made the choice to not build es5 build of the API because we though it's responsibility of an application to compile file with whatever tools they are using. Does building the embed worker in es5 is an exception ?

Reply:
Yeah the issue with the embedded worker is that it is usually not read by application bundlers (we even don't want to, as we fear those might bring syntax not authorized in a worker environment or introduce an outside scope).

I also brought another worker file in release notes, dist directory etc., as I guess applications would just use it directly without re-compiling it.

Reply:
Ok I see.